crashing while trying to access data folder

**Describe the bug**
crashes while trying ti access `/storage/emulated/0/Android/data`. the latest update `3.6.2` added support to view the folder on phones running Android 11 but i guess it doesn't work.

**Additional context**

* __App Name:__ Amaze File Manager
* __Package:__ com.amaze.filemanager
* __Version:__ 3.6.2
* __User Action:__ UI Error
* __Request:__ Application crash
* __OS:__ Linux Android 11 - 30
* __Device:__ gram
* __Model:__ POCO M2 Pro
* __Product:__ evolution_miatoll
<details><summary><b>Crash log </b></summary><p>

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)
Caused by: java.lang.NullPointerException
	at com.amaze.filemanager.utils.OTGUtil.getDocumentFiles(OTGUtil.kt:117)
	at com.amaze.filemanager.asynchronous.asynctasks.LoadFilesListTask.listDocumentFiles(LoadFilesListTask.java:570)
	at com.amaze.filemanager.asynchronous.asynctasks.LoadFilesListTask.doInBackground(LoadFilesListTask.java:209)
	at com.amaze.filemanager.asynchronous.asynctasks.LoadFilesListTask.doInBackground(LoadFilesListTask.java:69)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 3 more

```
</details>
<hr>

--- a/app/src/main/java/com/amaze/filemanager/utils/OTGUtil.kt
+++ b/app/src/main/java/com/amaze/filemanager/utils/OTGUtil.kt
@@ -98,6 +98,7 @@ object OTGUtil {
         fileFound: OnFileFound
     ) {
         var rootUri = DocumentFile.fromTreeUri(context, rootUriString)
+
         val parts: Array<String> = if (openMode == OpenMode.DOCUMENT_FILE) {
             path.substringAfter(rootUriString.toString())
                 .split("/", PATH_SEPARATOR_ENCODED).toTypedArray()
@@ -110,11 +111,16 @@ object OTGUtil {
             if (part == "otg:" || part == "" || part == "content:") continue
 
             // iterating through the required path to find the end point
-            rootUri = rootUri!!.findFile(part)
+            rootUri = rootUri?.findFile(part)
+        }
+
+        if (rootUri == null) {
+            Log.e(TAG, "Null DocumentFile listed!")
+            return
         }
 
         // we have the end point DocumentFile, list the files inside it and return
-        for (file in rootUri!!.listFiles()) {
+        for (file in rootUri.listFiles()) {
             if (file.exists()) {
                 var size: Long = 0
                 if (!file.isDirectory) size = file.length()

